cypress/integration/bundled-osd/check_sample_data.js

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
Is this testing existence of prior data instead off sample data. I think sample data shouldn't be factored in backwards compatibility tests since they are different indices and just for sample purposes.

Reply:
I think we could add this check_sample_data.js even though the indices names are diff. If cx add sample data in old version and then migrate, the dashboards page will look like this:
<img width="1516" alt="Screen Shot 2021-11-09 at 10 21 39 AM" src="https://user-images.githubusercontent.com/79961084/140982174-ef80ac76-9250-49a6-9a3d-261edcbd5f3b.png">

Even though the indices names are diff, cx can play around and all the saving filter and queries are kept in here. So I think it is necessary to test whether these indices can work. What do you think?